### DISCUSSION TITLE
shortcut key Volume up or down for focused application

### DISCUSSION THREAD

**Discussion by LightBeamOvHe** [File-New-Project/EarTrumpet#1573] [Ideas]
Hi,
I'm looking for an option to turn up or down the volume of a program that is focused. I can see there's that option for all peripherals, but not for a specific program.
My use case would be to lower the volume of my game, so I can hear a youtube video better, for example. Or the opposite, click on the browser, use whatever key shortcut, instead of using the UI.

Do you think it would be feasible ?

Anyway, thanks a lot for the work that's been done for this program. It's been serving me for many years now and is mandatory for my personal use on Windows. I've been kinda reluctant to ask for features, I feel like a choosing beggar of some sort lol, so I hope my approach doesn't sound too nonchalant.
Thanks again 😊

**Comment by RMRx** [comment]
I second this request, this is a feature I am truly missing in windows: muting, unmuting and adjusting the active app's volume per keybind, without having to open another window & click there. Especially entertainment apps in fullscreen mode like video or gaming tend to collide with active window changes.

[Nircmd](http://www.nirsoft.net/utils/nircmd.html)'s [setappvolume](http://nircmd.nirsoft.net/setappvolume.html), [changeappvolume](http://nircmd.nirsoft.net/changeappvolume.html) and [muteappvolume](http://nircmd.nirsoft.net/muteappvolume.html) offer something like that, but with a detour via shortcuts with hotkeys and command windows popping up, quite unelegant. 

Would be great if you could some day find the time for this.

**Reply by riverar** [reply]
If you install EarTrumpet dev builds (https://eartrumpet.app), you get EarTrumpet Actions which support this.

![image](https://github.com/user-attachments/assets/88831bee-3174-45e1-a92f-0d2af0e6bf84)

**Reply by RMRx** [reply]
Works great, tyvm. 
May I impertinently extend this feature request? Showing a volume OSD for that specific app would be the cherry on top.
